client/lib/analytics/ad-tracking/load-tracking-scripts.js

```diff
@@ -15,11 +15,11 @@ import {
 	OUTBRAIN_SCRIPT_URL,
 	PINTEREST_SCRIPT_URL,
 } from './constants';
-
-// Ensure setup has run.
-import './setup';
+import { setupAllTrackingScripts } from './setup';
```

Comment:
Given that the side effect is imminent during import, it should be enough to just import the entire module rather than a particular export from it.